package.json

```diff
@@ -15,7 +15,9 @@
 		"nohoist": [
 			"**/config",
 			"**/testarmada-magellan",
-			"**/testarmada-magellan-*"
+			"**/testarmada-magellan-*",
+			"calypso/@wordpress/interface",
+			"calypso/@wordpress/edit-post"
```

Comment:
These are required because https://github.com/WordPress/gutenberg/blob/master/packages/edit-post/src/style.scss#L1 does `@import "../../interface/src/style.scss";`, which assumes `@wordpress/edit-post` and `@wordpress/interface` will always be at the same level.

Unhoisting them ensures that `client/node_modules/@wordpress/edit-post` and `client/node_modules/@wordpress/interface` always exists.